src/main/java/uk/gov/justice/digital/hmpps/cmd/api/service/DryRunNotificationService.kt

```diff
@@ -37,21 +43,21 @@ class DryRunNotificationService(
 ) {
 
   @Transactional(propagation = Propagation.NEVER)
-  fun dryRunSendNotifications() {
+  fun sendNotifications() {
     val unprocessedNotifications = dryRunNotificationRepository.findAllByProcessedIsFalse()
-    log.info("dryRunSendNotifications: Sending notifications, found: ${unprocessedNotifications.size}")
+    log.info("dryRun sendNotifications: Sending notifications, found: ${unprocessedNotifications.size}")
     unprocessedNotifications.groupBy { it.quantumId }
       .forEach { group ->
         try {
-          // NO! sendNotification(group.key, group.value)
+          sendNotification(group.key, group.value)
```

Comment:
ther crucial line. Everything else is just copied over from the existing Notification code